CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
-# NEXT_VERSION
+# v2.0.0alpha4 (Unreleased)
 
-- Update clip step in lonlat.py to solve clipping issue on lat layers
+- Resolve issue with clipping of latitude reference layers
```

Comment:
Let's add a note about using JPEG compression for the background layer here.

Reply:
63fad2a